docs/api/v1.21.md

```diff
@@ -2836,7 +2836,12 @@ Content-Type: application/json
 **JSON parameters**:
 
 - **Name** - The new network's name. this is a mandatory field
-- **CheckDuplicate** - Requests daemon to check for networks with same name. Defaults to `false`
+- **CheckDuplicate** - Requests daemon to check for networks with same name. Defaults to `false`.
+    Since Network is primarily keyed based on a random ID and not on the name, 
+    and network name is strictly a user-friendly alias to the network which is uniquely identified using ID, 
+    there is no guaranteed way to check for duplicates across a cluster of docker hosts. 
```

Comment:
It is from the comment of #18864, my understanding is that since multi-hosts may use a K/V store to store network details and keyed these networks with their ID, not name.  So there is no guaranteed way to check for duplicated network via name. 
Keep improving, maybe miss some other important details, open for input.